src/nvim/syntax.c

```diff
@@ -1571,36 +1622,33 @@ syn_finish_line(
   return false;
 }
 
-/*
- * Return highlight attributes for next character.
- * Must first call syntax_start() once for the line.
- * "col" is normally 0 for the first use in a line, and increments by one each
- * time.  It's allowed to skip characters and to stop before the end of the
- * line.  But only a "col" after a previously used column is allowed.
- * When "can_spell" is not NULL set it to TRUE when spell-checking should be
- * done.
- */
-int
-get_syntax_attr(
-    const colnr_T col,
-    bool *const can_spell,
-    const bool keep_state           // keep state of char at "col"
-)
+/// @return  highlight attributes for next character.
```

Comment:
looks like a bad search-replace. Would change to: 

```suggestion
/// Gets highlight attributes for next character.
```

and maybe also add `@return` blurb at the end